sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/api/Azure.ResourceManager.NeonPostgres.netstandard2.0.cs

```diff
@@ -0,0 +1,254 @@
+        Azure.ResourceManager.NeonPostgres.Models.PartnerOrganizationProperties System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.PartnerOrganizationProperties>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.PartnerOrganizationProperties>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.PartnerOrganizationProperties>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ResourceProvisioningState : System.IEquatable<Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ResourceProvisioningState(string value) { throw null; }
+        public static Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState Canceled { get { throw null; } }
+        public static Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState Failed { get { throw null; } }
+        public static Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState Succeeded { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState left, Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState left, Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class SingleSignOnProperties : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NeonPostgres.Models.SingleSignOnProperties>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.SingleSignOnProperties>
```

Comment:
The current name is too generic. Please rename it to something more specific and appropriate, such as `NeonSingleSignOnProperties`.